src/Components/GraduatesList/Graduated.jsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import { Checkbox, Table } from "semantic-ui-react";
+
+function Graduated(props) {
+	return (
```

Comment:
Try of use an indentation of 2 spaces.

Reply:
Agreed, please set your IDE for 2 spaces indetation.

Reply:
Thaks, I did that